### PR TITLE
Sb 22307, SB-22290, SB-22289 Fix

### PR DESCRIPTION
**Pull request by amit-tarento**

### CODE REVIEW

**Review by reshmi-nair — APPROVED** [review]

File: all-actors/src/main/java/org/sunbird/notification/dispatcher/impl/FCMNotificationDispatcher.java [hunk 1]
```diff
@@ -206,7 +205,7 @@ private String getRequestHashed(Map<String, Object> request) {
     try {
       val = DataHash.getHashed(mapper.writeValueAsString(request));
     } catch (Exception e) {
-      logger.error("exception occured during hash of request data:" + e.getMessage());
+      logger.error("exception occured during hash of request data:" + e.getMessage(), e);
```
Comment (Hari-stackroute) [inline all-actors/src/main/java/org/sunbird/notification/dispatcher/impl/FCMNotificationDispatcher.java:208]: context is missing.

Comment (amit-tarento) [reply all-actors/src/main/java/org/sunbird/notification/dispatcher/impl/FCMNotificationDispatcher.java]: Done

File: pom.xml [hunk 2]
```diff
@@ -30,6 +28,40 @@
         <powermock.version>2.0.2</powermock.version>
         <httpClient.version>4.5.1</httpClient.version>
     </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.7.5.201505241946</version>
```
Comment (Hari-stackroute) [inline pom.xml:36]: In other repos we used version=0.8.5, is it ok to use this?

Comment (amit-tarento) [reply pom.xml:36]: As of now i didnt see any issue

**Review by Hari-stackroute — APPROVED** [review]

**Pull request merged**